package.json

```diff
@@ -19,6 +19,7 @@
     "@uiw/codemirror-themes-all": "^4.21.11",
     "@uiw/react-codemirror": "^4.21.11",
     "autoprefixer": "10.4.15",
+    "axios": "^1.5.0",
```

Comment:
Why did we install axios? I don't think there's a need for this package.

Reply:
I have removed this.